strawberryfields/apps/qchem/dynamics.py

```diff
@@ -212,3 +215,36 @@ def sample_fock(
         s.append(eng.run(prog).samples[0].tolist())
 
     return s
+
+
+def prob(samples: list, excited_state: list) -> float:
+
+    r"""Generate probability of observing a Fock state.
+
+    **Example usage:**
+
+    >>> excited_state = [0, 2]
+    >>> samples = [[0, 2], [1, 1], [0, 2], [2, 0], [1, 1], [0, 2], [1, 1], [1, 1], [1, 1], [0, 2]]
+    >>> prob(samples, excited_state)
+    0.4
```

Comment:
```suggestion
    >>> samples = [[0, 2], [1, 1], [0, 2], [2, 0], [1, 1], [0, 2], [1, 1], [1, 1], [1, 1]]
    >>> prob(samples, excited_state)
    0.3333333333333333
```
This is picky, but just to prevent the user having to scroll:
![image](https://user-images.githubusercontent.com/49409390/85063516-429d0500-b178-11ea-9e41-7753ac58b2e1.png)